### DISCUSSION TITLE
Problem with README.exim.md

### DISCUSSION THREAD

**Discussion by bjornblomqvist** [roehling/postsrsd#98] [Ideas]
The example in README.exim.md does not work as is. The return_path set would always include a  space at the end and would get rejected by the receiving smtp server.

`Return-Path: <SRS0=0uXk=2F=another-dummy.net=jane.doe@dummy-host.com >`

My fix was to remove the line breaks in the configuration like seen below.

```
return_path = ${if and { \
             {!match_ip{$sender_host_address}{:@[]}} \
             {!def:authenticated_id} \
             {!match_address {$sender_address} { : *@+local_domains : *@+virtual_domains : SRS_DOMAIN}} \
     } \
     {${if match {${readsocket{inet:localhost:10001}{get $sender_address}{3s}}}{\N^200 (.+)\N}{$1}fail}}fail}
```

Unfortunately I am not very good at exim4 configuration so I cant say where in my change the fix is.

I am using Exim 4.92.

**Comment by roehling** [comment]
I'm not using exim either, so I'm glad to take pull requests with a known good configuration, but I cannot verify (or fix) it myself.
